Automattic/wp-calypso
# On /plans, use default intervalType based on a current plan

This is related to multiyear subscriptions project: p9jf6J-eR-p2

"My upgrades" page will need to display 2-year upgrades if user is currently on a 2-year plan. This PR passes `intervalType` prop with value "2yearly" for 2-year plans, "yearly" for 1-year plans, and null for monthly plans - so far it wasn't set to anything particular and I don't want to affect jetpack plans page.

Test plan:
* Run unit tests
* Point `public-api.wordpress.com` to your sandbox
* Enable store sandbox and bump products cache version
* Run `ENABLE_FEATURES=upgrades/2-year-plans npm run start `
* Go to `/plans` and confirm it does not blow up
* Run `npm run start `
* Go to `/plans` and confirm it does not blow up
* Actual update to this screen to reflect 2-year plans will come in another PR
